content/en/docs/ops/ambient/usage/ztunnel/_index.md

```diff
@@ -0,0 +1,498 @@
+
+
+{{< text bash >}}
+$ kubectl exec -it deploy/sleep -n ambient-demo  -- curl httpbin:8000 -s | grep title -m 1
+{{< /text >}}
+{{< text syntax=plain snip_id=none >}}
+    <title>httpbin.org</title>
+{{< /text >}}
+
+In this case, the traffic traversed an enecyrpted ztunnel. In the next section we look at how to check that traffic is using the ztunnel and some monitoring options. 
+
+## Monitoring the ztunnel data path
+
+
+### Verifying ztunnel traffic logs
+
+Let us send some traffic from a client `sleep` pod to the `httpbin` service.
+
+{{< text bash >}}
+$ kubectl -n ambient-demo exec deploy/sleep -- sh -c 'for i in $(seq 1 10); do curl -s -I http://httpbin:8000/; done'
+{{< /text >}}
+The response displayed confirms the client pod receives responses from the service. 
+{{< text syntax=plain snip_id=none >}}
+HTTP/1.1 200 OK
+Server: gunicorn/19.9.0
+--snip--
+{{< /text >}}
+
+Now lets check logs of the `ztunnel` pods to confirm the traffic was sent over the `ztunnel`.
+
+{{< text bash >}}
+$ kubectl -n istio-system logs -l app=ztunnel | egrep "inbound|outbound"
+{{< /text >}}
+{{< text syntax=plain snip_id=none >}}
+2023-08-14T09:15:46.542651Z  INFO outbound{id=7d344076d398339f1e51a74803d6c854}: ztunnel::proxy::outbound: proxying to 10.240.2.10:80 using node local fast path
+2023-08-14T09:15:46.542882Z  INFO outbound{id=7d344076d398339f1e51a74803d6c854}: ztunnel::proxy::outbound: complete dur=269.272µs
+--snip--
+{{< /text >}}
+
+These log messages confirm the traffic indeed used the `ztunnel` proxy and traffic redirection between pods and the proxy is likely working correctly. Additional fine grained monitoring can be done by checking logs on the specific `ztunnel` proxy instances that are on the same nodes as the source and destination pods of traffic.  If these logs are not seen, then a possibility is that traffic redirection may not be working correctly and the next section has some information on checking for that scenario. Note that with `ambient` traffic always traverses the `ztunnel` pod even when the source and destination of the traffic are on the same compute node. 
+
+### Checking for ztunnel traffic redirection
+In case the traffic logs for the `ztunnel` proxy do not indicate forwarding of traffic, you can check whether traffic redirection has been set up correctly. In this section we do not go into the full details of how traffic redirection is performed, but we provide some quick pointers to check.
```

Comment:
I'm of a mind that all the `iptables` and `ipset` stuff here should be moved out into an "advanced debugging" guide, that lives in the devwiki/dev README - it's tightly tied to the current impl, and it's far more than we need to ask users in order to get a good bug report, and far more than what users need to validate that ambient is working in their cluster.

The goal of the docs here should be equipping people to know where to check that that ztunnel is capturing traffic, and to know where to go to collect the correct logs (`ztunnel` and `istio-cni`) if something isn't working.

Reply:
okay, will remove the subsections related to traffic redirection and add a note that this content will be in a separate guide.